services/search/pkg/command/index.go

```diff
@@ -31,7 +31,7 @@ func Index(cfg *config.Config) *cli.Command {
 				Name:     "user",
 				Aliases:  []string{"u"},
 				Required: true,
-				Usage:    "the username of the user tha shall be used to access the files",
+				Usage:    "the id of the user tha shall be used to access the files",
```

Comment:
```suggestion
				Usage:    "the id of the user that shall be used to access the files",
```

Reply:
Also, there's another typo in https://github.com/owncloud/ocis/blob/068709dad37f10aedbdf39a42dee71ec64c51b1c/services/search/pkg/command/index.go#L20

Reply:
thanks 😗